CHANGES.rst

```diff
@@ -277,6 +277,8 @@ tso_photometry
 - Fix a bug in the computation of integration time stamps when the INT_TIMES
   table is not available. [#5318]
 
+- Fixed the flux units in the output photometry catalog. [#5529]
```

Comment:
This needs to be moved up into the 0.17.2 (unreleased) section of the log file. Create a tso_photometry subsection up there (e.g. up around line 170).

Reply:
done